test/diagnostics/package-name-diagnostics.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/split_file.py -o %t %s
```

Comment:
Is `split_file` needed here? It looks to me like there are not multiple inputs. If it is needed, I believe this can be simplified to:
```suggestion
// RUN: split-file %s %t
```

Reply:
Copied from another test; will remove